examples/windows/qt-window-app/backtrace.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <string_view>
+
+namespace backtrace
+{
+bool initialize_crashpad(std::string_view url);

```

Comment:
should we add tabs here?

Reply:
No, namespaces are not indented in any major C++ coding styles including Crashpad's.